--- mosquitto-1.4.10/config.mk	2016-08-17 13:05:14.000000000 +0200
+++ config.mk	2016-11-08 16:00:55.909071119 +0100
@@ -58,14 +58,14 @@
 WITH_SYS_TREE:=yes
 
 # Build with SRV lookup support.
-WITH_SRV:=yes
+WITH_SRV:=no
 
 # Build using libuuid for clientid generation (Linux only - please report if
 # supported on your platform).
 WITH_UUID:=yes
 
 # Build with websockets support on the broker.
-WITH_WEBSOCKETS:=no
+WITH_WEBSOCKETS:=yes
 
 # Use elliptic keys in broker
 WITH_EC:=yes
